R/github-actions.R

```diff
@@ -251,6 +251,12 @@ use_tidy_github_actions <- function(ref = NULL) {
   use_coverage(repo_spec = repo_spec)
   use_github_action("test-coverage", ref = ref)
 
+  if (!uses_air()) {
+    ui_bullets(c(
+      "!" = "Can't find an {.file air.toml} file. Do you need to run
+             {.run [use_air()](usethis::use_air())}?"
+    ))
+  }
   use_github_action(
     url = "https://github.com/posit-dev/setup-air/blob/main/examples/format-suggest.yaml"
   )
```

Comment:
Oh interesting I was originally suggesting that we'd only ever add the `format-suggest.yaml` if we detected usage of air, like

```r
if (uses_air()) {
  use_github_action(".../format-suggest.yaml")
}
```

I don't mind being a little pushy about using air though

Reply:
I think it's ok to be pushy in `use_tidy_github_actions()`, since the only folks who are pseudo-required to do this is us.